Schema changes that are permitted with compression (#663)
timescale/docs.timescale.com-content: api.md

need to backfill or update data in a compressed chunk you will need to
decompress the chunk(s) first.

> Starting with TimescaleDB release 2.1, users have the ability to modify metadata
> for hypertables that have compressed chunks.
> Specifically,[you can add columns to a hypertable, and rename existing columns](#compression-schema-changes).

Reviewer: ```suggestion
Starting with TimescaleDB 2.1, users have the ability to modify the schema
of hypertables that have compressed chunks.
Specifically, [you can add columns to and rename existing columns of such compressed hypertables](#compression-schema-changes).
```
